Calorie Tracker/Calorie Tracker/ViewControllers/CalorieTableViewController.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  CalorieTableViewController.swift
+//  Calorie Tracker
+//
+//  Created by Alex Thompson on 2/22/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+import CoreData
+import SwiftChart
+
+class CalorieTableViewController: UITableViewController {
+    @IBOutlet private weak var chartView: Chart!
+    
+    
+    
+    let calorieController = CalorieController()
+    
+    lazy var fetchedResultController: NSFetchedResultsController<Calorie> = {
+        
+        let fetchRequest: NSFetchRequest<Calorie> = Calorie.fetchRequest()
+        fetchRequest.sortDescriptors = [
+            NSSortDescriptor(key: "timestamp", ascending: true)
+        ]
+        let moc = CoreDataStack.shared.mainContext
+        let frc = NSFetchedResultsController(fetchRequest: fetchRequest, managedObjectContext: moc, sectionNameKeyPath: "calorie", cacheName: nil)
```

Comment:
Do spell out things, especially in longer methods. It makes code easier to read.